### ISSUE TITLE
NVDA 2018 1.1 have to "restart" NVDA or "refresh" webpage to arrow or use browser shortcuts "i.e. h for headers" especially after search.

### ISSUE THREAD

**Issue by ythompson9** [nvaccess/nvda#8135]
1. go to startpage.com; walmart.com; homeshop.kroger.com
2.  Input text in the search box.
3.  Try to arrow down the page, use "e for edit box" navigation, "h for header" navigation, etc. they do not 
work.### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### E
xpected behavior:
> Tell us what should happen.
NVDA 2018 1.1 should perform like NVDA 2018 1 and allow you to arrow down the page, or use shortcut keys to access elements within the page.
### Actual behavior:
> Tell us what happens instead.
You are unable to use arrow keys or shortcut keys unless you "restart NVDA" or refresh the webpage.
### System configuration:
NVDA version:
> Example: next-14027,c80e529f
Windows 10
Mozilla esr
NVDA 2018 1.1
NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by michaelDCurran** [comment]
Does the problem continue to occur when you restart your web browser and 
or  the machine?


We are certainly aware of a bug where updating NVDA causes an already 
open browser to no longer show page updates with NVDA.

**Comment by gdata1** [comment]
Hello. I'm not quite sure what the problem you're talking about.
A similar problem occurs on some sites in Microsoft edge. When you go to some Web sites, Edge starts reloading the page.

**Comment by Brian1Gaff** [comment]
I sent this on a personal mail, but on windows 7, the new update cannot be 
downloaded from within nvda. it will however see it from older copies like 
2017.3.
 Brian

**Comment by ehollig** [comment]
@ythompson9, I can not replicate your issue. I performed the following steps:
1. Go to startpage.com in Firefox.
2. Navigate to the first edit field on the page by pressing E and activate browse mode by pressing enter.
3. Enter some text in the field and press enter to submit the search.
4. Navigate down the page by headers using the H key, or to the next edit field by pressing E

As @michaelDCurran said in https://github.com/nvaccess/nvda/issues/8135#issuecomment-377652951 there is an issue where updating NVDA causes an already
open browser to no longer show page updates with NVDA.
Have you tried running the "COM Registration Fixing tool" in NVDA Tools to see if this helps your issue?

**Comment by ythompson9** [comment]
No, I have not, was not aware of that tool, thank you for this. I bet
this fixes it.  Love the program, so much better than the other screen
reader on the market.
Thank you for your assistance.

**Comment by ehollig** [comment]
I am closing this as worksforme. If this is still an issue, we can reopen.
